datagateway_api/src/resources/search_api_endpoints.py

```diff
@@ -31,7 +34,27 @@ class Endpoint(Resource):
         def get(self):
             filters = get_filters_from_query_string("search_api", entity_name)
             log.debug("Filters: %s", filters)
-            return get_search(entity_name, filters), 200
+            log.debug("entity_name: %s", entity_name)
+
+            if not is_query_parameter_enabled(filters):
+                entities = get_search(entity_name, filters)
+                return entities, 200
+            else:
+                query = get_search_api_query_filter_list(filters)[0].value
+
+                entities = get_search(
+                    entity_name,
+                    filters,
+                    "LOWER(o.summary) like '%" + query.lower() + "%'",
```

Comment:
Looking at this line and line 56 (where you pass `"investigations"` to `get_score()`), I guess this only works on /documents for now? Have you got a plan of how to make it work for the other endpoints?

Reply:
You are right and honestly I do not know. There are several reason:
1) I focused on making https://data.panosc.eu/ to work. Surprisingly I did discover that only use `/documents` endpoint. So, I did not consider to add something that is not used
2) When I tried to use the scoring app with the datasets, it did not work when calculating the score (my guess is because of the number of datasets). I created an issue: 
https://github.com/panosc-eu/panosc-search-scoring/issues/9

So, yes, it might be needed in the future but it very uncertain and first we would need the scoring app to work at the level of datasets

Reply:
We were slightly concerned about the way the score calculation behaves and that it might cause performance issues with large volumes of data. It would be good to confirm (from someone who wrote the scoring software perhaps?) which endpoint(s) the scoring needs to work on.